tests/test_file_watcher_integration.py

```diff
@@ -27,7 +27,7 @@ async def test_file_watcher(tmp_path: pathlib.Path) -> None:
     expected_number_of_writes = 3
 
     file_watcher = FileWatcher(paths=[str(tmp_path)])
-    timer = Timer.timeout(timedelta(seconds=0.1))
+    timer = Timer(timedelta(seconds=0.1), SkipMissedAndDrift())
```

Comment:
The policies can have options, and `SkipMissedAndDrift()` have one ([`delay_tolerance`](https://frequenz-floss.github.io/frequenz-channels-python/v1.0-dev/reference/frequenz/channels/timer/#frequenz.channels.timer.SkipMissedAndDrift.delay_tolerance)). I think if we provide fixed instances with fixed options we'll eventually need another PR to remove them because they are still hiding the complexity :)